docs/source/_json/workflow_get.json

```diff
@@ -0,0 +1,27 @@
+GET /plone/front-page/workflow
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+{
+  "history": [
+    {
+      "action": null, 
+      "actor": "test_user_1_", 
+      "comments": "", 
+      "review_state": "private", 
```

Comment:
The current workflow state will also be included in the object's GET response.
Regarding the workflow endpoint, i guess getting the complete history is enough. What else would you expect?